tests/integration_tests/performance/test_block_performance.py

```diff
@@ -279,7 +278,8 @@ def test_block_performance(
             )
             st_cons = st.consumer.LambdaConsumer(
                 metadata_provider=DictMetadataProvider(
-                    CONFIG["measurements"], BlockBaselinesProvider(env_id, fio_id)
+                    CONFIG["measurements"],
+                    BlockBaselinesProvider(env_id, fio_id, CONFIG),
```

Comment:
Is there any reason why we use `CONFIG` here different from other perf tests?

Reply:
All other tests have been updated to use pytest-level parameterization instead of reading things such as "what numbers of vcpus to test" from the baseline file in https://github.com/firecracker-microvm/firecracker/pull/3736. For the block test, I couldn't do that, since it changes the performance characteristics in a way that'll require baseline regathering (since currently, subsequent block tests reuse the same VM, but with pytest parameterization, each test will get its own VM). That's why the change has been pushed back for the block test, and why we use the `CONFIG` dict differently here

Reply:
ah sorry, I missed the commit message. thanks for your explanation.